dotnet/Services/ProductReviewService.cs

```diff
@@ -69,9 +70,16 @@ public async Task<bool> DeleteLegacyReview(int[] ids, string productId = null)
             return retval;
         }
 
-        public async Task<Review> EditReview(Review review)
+        public async Task<(Review, HttpStatusCode)> EditReview(Review review)
         {
 
+            HttpStatusCode isValidAuthUser = await IsValidAuthUser();
+            
+            if (isValidAuthUser != HttpStatusCode.OK)
+            {
+                return (null, isValidAuthUser);
+            }
+
```

Comment:
Since this and most of the other methods are shared by GraphQL and the REST API, and the REST API route handlers already validate the user's authentication, I would suggest moving this to the GraphQL resolver. I'm also not sure if `ctx.vtex.adminuserauthtoken` is populated when the REST API is used. What do you think @btalma ?

Reply:
This also might be why Cypress is failing

Reply:
That is a good question.  I do not know if adminuserauthtoken is populated when using the API.  Most likely it is not.  I agree that the check should be done in GraphQL.

Reply:
Made the change, auth check now on resolver side